src/Reflection/StringCast/ReflectionConstantStringCast.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Reflection\StringCast;
+
+use Roave\BetterReflection\Reflection\ReflectionConstant;
+use function gettype;
+use function sprintf;
+
+/**
+ * @internal
+ */
+final class ReflectionConstantStringCast
```

Comment:
Reading this class doesn't make it clear what it is to be used for: can you add a sentence explaining that it is supporting/utility?

Reply:
Added.